### PR TITLE
Migrated the deprecated pick image method to get Image.(Bug fix)

### PR DESCRIPTION
**Pull request by i-anuragmishra**
**What kind of change does this PR introduce?**
**This PR Resolved the Issue of #431 . And MIgrates the the deprecated method to use "picker.getimage" which works with newer flutter version.**

This is a Bugfix.

**Did you add tests for your changes?**
no.

**If relevant, did you update the documentation?**
No, Documentation need not be updated.

**Summary**
Since the Pick image method was deprecated by flutter it was creating a bug in camera. so it has been migrated to get image method.

It was creating a Bug , since the method is not supported in flutter so it needs to be migrated toa supported method.

Issue that this PR resolves is : #431 

**Does this PR introduce a breaking change?**
No, only a Bug fix

### CODE REVIEW

File: lib/views/pages/login_signup/register_form.dart [hunk 1]
```diff
@@ -131,18 +133,21 @@ class RegisterFormState extends State<RegisterForm> {
...
+              )));
     }
   }
 
   //get image using camera
+  final _picker = ImagePicker();
   _imgFromCamera() async {
-    File image = await ImagePicker.pickImage(
-        source: ImageSource.camera, imageQuality: 50);
+    PickedFile _image =
```
Comment (Sagar2366) [inline lib/views/pages/login_signup/register_form.dart:146]: This is only change required to be part of this PR, please remove all unnecessary changes.

Comment (i-anuragmishra) [reply lib/views/pages/login_signup/register_form.dart:146]: okay wil close this PR and push from newer branch

**Comment by i-anuragmishra** [comment]
Closing this PR becouse of Additional Commits
